alf/algorithms/rlpd_algorithm.py

```diff
@@ -0,0 +1,213 @@
+                 critic_loss_ctor=None,
+                 target_entropy=None,
+                 prior_actor_ctor=None,
+                 target_kld_per_dim=3.,
+                 initial_log_alpha=0.0,
+                 max_log_alpha=None,
+                 target_update_tau: Union[float, Scheduler] = 0.05,
+                 target_update_period: Union[int, Scheduler] = 1,
+                 parameter_reset_period: Union[int, Scheduler] = -1,
+                 dqda_clipping=None,
+                 actor_optimizer=None,
+                 critic_optimizer=None,
+                 alpha_optimizer=None,
+                 checkpoint=None,
+                 debug_summaries=False,
+                 name="RlpdAlgorithm"):
+        # **kwargs):
+        """
+        Refer to SacAlgorithm for more details for kwargs
+
+        Args:
+            name (str): The name of this algorithm.
+            num_critic_targets (int): Number of sampled subset of target critics
+                for computing TD target in critic training.
+        """
+        super().__init__(
+            observation_spec=observation_spec,
+            action_spec=action_spec,
+            reward_spec=reward_spec,
+            actor_network_cls=actor_network_cls,
+            critic_network_cls=critic_network_cls,
+            repr_alg_ctor=repr_alg_ctor,
+            reward_weights=reward_weights,
+            train_eps_greedy=train_eps_greedy,
+            epsilon_greedy=epsilon_greedy,
+            use_entropy_reward=use_entropy_reward,
+            normalize_entropy_reward=normalize_entropy_reward,
+            calculate_priority=calculate_priority,
+            num_critic_replicas=num_critic_replicas,
+            env=env,
+            config=config,
+            critic_loss_ctor=critic_loss_ctor,
+            target_entropy=target_entropy,
+            prior_actor_ctor=prior_actor_ctor,
+            target_kld_per_dim=target_kld_per_dim,
+            initial_log_alpha=initial_log_alpha,
+            max_log_alpha=max_log_alpha,
+            target_update_tau=target_update_tau,
+            target_update_period=target_update_period,
+            parameter_reset_period=parameter_reset_period,
+            dqda_clipping=dqda_clipping,
+            actor_optimizer=actor_optimizer,
+            critic_optimizer=critic_optimizer,
+            alpha_optimizer=alpha_optimizer,
+            checkpoint=checkpoint,
+            debug_summaries=debug_summaries,
+            name=name)
+
+        assert self._act_type == ActionType.Continuous, (
+            "RLPD algorithm only supports continuous action spaces.")
+
+        self._num_critic_targets = num_critic_targets
+
+    def _compute_critics(self,
+                         critic_net,
+                         observation,
+                         action,
+                         critics_state,
+                         replica_consensus='mean',
+                         sample_subset=False,
```

Comment:
can add some comments to ``sample_subset`` argument, and the reason using ``True`` for target critics and ``False`` for critics.

Reply:
Updated.